Fix:

In file included from modules/speller/default/readonly_ws.cpp:51:
modules/speller/default/vector_hash-t.hpp:186:43: error: no member named 'e' in 'VectorHashTable<Parms>'
  186 |     for (iterator i = begin(); i != this->e; ++i, ++this->_size);
      |                                     ~~~~  ^
modules/speller/default/vector_hash-t.hpp:186:59: error: no member named '_size' in 'VectorHashTable<Parms>'; did you mean 'size_'?
  186 |     for (iterator i = begin(); i != this->e; ++i, ++this->_size);
      |                                                           ^~~~~
      |                                                           size_

https://github.com/GNUAspell/aspell/commit/ee6cbb12ff36a1e6618d7388a78dd4e0a2b44041
--- modules/speller/default/vector_hash-t.hpp.orig
+++ modules/speller/default/vector_hash-t.hpp
@@ -183,7 +183,7 @@ namespace aspeller {
   template<class Parms>
   void VectorHashTable<Parms>::recalc_size() {
     size_ = 0;
-    for (iterator i = begin(); i != this->e; ++i, ++this->_size);
+    for (iterator i = begin(), e = end(); i != e; ++i, ++size_);
   }
 
 }
